__frzr-deploy

```diff
@@ -181,37 +182,47 @@ main() {
 		stdbuf -oL tr '\r' '\n' | grep --line-buffered -oP '[0-9]*+(?=.[0-9])' | grep --line-buffered -v '100' | \
 		whiptail --gauge "Downloading system image (${NAME})" 10 50 0
 	fi
-	
+
 	CHECKSUM2=`sha256sum "${IMG_FILE}" | cut -d' ' -f 1`
 	if [ "$CHECKSUM" != "$CHECKSUM2" ]; then
 		rm -f "${IMG_FILE}"
 		echo "checksum does not match; aborting"
 		exit 1
 	fi
-	
+
 	if [ -z ${SHOW_UI} ]; then
 		echo "installing ${NAME}..."
 	else
 		whiptail --infobox "Extracting and installing system image (${NAME}). This may take some time." 10 50
 	fi
-	
+
 	tar xfO ${IMG_FILE} | btrfs receive --quiet ${DEPLOY_PATH}
 	mkdir -p ${MOUNT_PATH}/boot/${NAME}
 	cp ${SUBVOL}/boot/vmlinuz-linux ${MOUNT_PATH}/boot/${NAME}
 	cp ${SUBVOL}/boot/initramfs-linux.img ${MOUNT_PATH}/boot/${NAME}
 
-	get_syslinux_cfg ${NAME} ${PREFIX} > ${BOOT_CFG}
-	
+	ADDITIONAL_INITRD=""
```

Comment:
Shouldn't we include both AMD and INTEL? What if someone has ChimeraOS installed on an external device and moves between computers?